tests/sacado/ad_number_traits_02c.output.1

```diff
@@ -0,0 +1,63 @@
+
```

Comment:
This alternate output is valid if that compiler error related to `std::complex<Sacado::Rad::ADvar>` is hit.

Reply:
you can name the file as `....ouptut.rad_advar_error` or something so that you know why it's there

Reply:
Thanks, I wasn't aware that we could label alternate outputs in this manner.